src/diapysef/diapysef/conversions.py

```diff
@@ -129,16 +129,20 @@ def align_im(msms_irt, ms, runs, im_alignment):
     return(ms)
 
 def reformat_mods(data, column):
-    data[column] = data[column].str.replace('_', '')
     data[column] = data[column].str.replace("(ox)","Oxidation")
     data[column] = data[column].str.replace("(ph)","Phospho")
     data[column] = data[column].str.replace("C","C(Carbamidomethyl)")
     data[column] = data[column].str.replace("(ac)","Acetylation")
+    data[column] = data[column].str.replace("_\\(ly\\)", ".(heavy_channel:modification_lysine)")
+    data[column] = data[column].str.replace("_\\(ar\\)", ".(heavy_channel:modification_arginine)")
```

Comment:
@klannieha does this actually work? Have you tested this? I tried this and it doesnt work for me:

```
>>> pyopenms.AASequence.fromString("PEPTIDEK.(heavy_channel:modification_lysine)")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyopenms/pyopenms_7.pyx", line 113, in pyopenms.pyopenms_7.__static_AASequence_fromString
RuntimeError: the element 'heavy_channel:modification_lysine' could not be found
```

while this actually works:

```
>>> pyopenms.AASequence.fromString("PEPTIDER(UniMod:267)")
<pyopenms.pyopenms_7.AASequence object at 0x7f1c302598b0>
>>> pyopenms.AASequence.fromString("PEPTIDEK(UniMod:259)")
<pyopenms.pyopenms_7.AASequence object at 0x7f1c302597b0>
```

so I am confused, why not use UniMod here?

I think the correct way to do it would be to either use UniMod or use the OpenMS notation:

```
>>> pyopenms.AASequence.fromString("PEPTIDEK(UniMod:259)").toString()
'PEPTIDEK(Label:13C(6)15N(2))'
>>> pyopenms.AASequence.fromString("PEPTIDER(UniMod:267)").toString()
'PEPTIDER(Label:13C(6)15N(4))'
```

Reply:
I thought it would be the same with the openms SILAC labels, I can change them to just the UniMod